go.sum

```diff
@@ -959,6 +959,8 @@ github.com/urfave/cli v1.20.0/go.mod h1:70zkFmudgCuE/ngEzBv17Jvp/497gISqfk5gWijb
 github.com/urfave/cli v1.22.1/go.mod h1:Gos4lmkARVdJ6EkW0WaNv/tZAAMe9V7XWyB60NtXRu0=
 github.com/vektah/gqlparser v1.1.2/go.mod h1:1ycwN7Ij5njmMkPPAOaRFY4rET2Enx7IkVv3vaXspKw=
 github.com/xanzy/go-gitlab v0.15.0/go.mod h1:8zdQa/ri1dfn8eS3Ir1SyfvOKlw7WBJ8DVThkpGiXrs=
+github.com/xcoulon/toolchain-common v0.0.0-20200902135505-09e93be9559a h1:9ZmPejkMV67l9bPhvzMCBy0oyMiudstNdYHAs0AyLxw=
```

Comment:
Friendly reminder 🙂

Reply:
oh good catch @rajivnathan! I thought I ran the `go mod tidy` command, but I guess not 🤦‍♂️

Reply:
done in https://github.com/codeready-toolchain/toolchain-e2e/pull/156/commits/8f7470fd09c891231e3bbb90bef5685fe131f2ae